packages/suite/src/utils/suite/dom.ts

```diff
@@ -50,6 +50,9 @@ export const useKeyPress = (targetKey: string) => {
 
 export const copyToClipboard = (value: string, parent: HTMLDivElement | HTMLPreElement | null) => {
     try {
+        if (value.length <= 0) {
+            return false;
```

Comment:
you should throw Error instead of return false, error will be catch in lower block

Reply:
throwing an error and then in the catch block return error.message as a string is anti-pattern, but ok, i rewrote